JS/Lesson30/tugba/my-app/src/components/Ex10/index.js

```diff
@@ -0,0 +1,22 @@
+import {useState} from 'react';
+
+export const Ex10 = () => {
+  // useState hook
+  const [clickCount, setClickCount] = useState(0);
+
+  function handleClick () {
```

Comment:
`event.detail` property would give you the information about how many times the button was clicked.

so the fucntion would be modified like this:
```
function handleClick (event) {
  if (event.detail === 2) alert("double-clicked");
}
```